Mathlib/LinearAlgebra/Dimension/FreeAndStrongRankCondition.lean

```diff
@@ -206,7 +206,6 @@ theorem finrank_eq_one_iff' [Module.Free K V] :
   exact toNat_eq_iff one_ne_zero
 #align finrank_eq_one_iff' finrank_eq_one_iff'
 
--- Not sure why this aren't found automatically.
```

Comment:
This referred to two `apply_instance` calls in the mathlib3 version of the previous proof, which have since been refactored away